src/vs/workbench/browser/parts/notifications/notificationsCommands.ts

```diff
@@ -137,6 +142,24 @@ export function registerNotificationCommands(center: INotificationsCenterControl
 		}
 	});
 
+	// Accept Primary Action
+	KeybindingsRegistry.registerCommandAndKeybindingRule({
+		id: ACCEPT_PRIMARY_ACTION_NOTIFICATION,
+		weight: KeybindingWeight.WorkbenchContrib,
+		when: ContextKeyExpr.and(NotificationsToastsVisibleContext, CONTEXT_ACCESSIBILITY_MODE_ENABLED),
+		primary: KeyMod.CtrlCmd | KeyCode.Enter,
+		handler: (accessor) => {
+			const actionRunner = accessor.get(IInstantiationService).createInstance(NotificationActionRunner);
+			const notification = model.notifications[0];
+			const primaryAction = notification?.actions?.primary;
+			if (!primaryAction?.length) {
+				return;
+			}
+			actionRunner.run(primaryAction[0], notification);
+			notification.close();
```

Comment:
Why do we still need to `close`?

Reply:
Because the `actionRunner` in the other file overrides the default `run` of `actionRunner` to `close`

Reply:
https://github.com/microsoft/vscode/blob/eb1129be5da436ed44d2c135ffc856fad8b7c5df/src/vs/workbench/browser/parts/notifications/notificationsViewer.ts#L461-L472